src/ingest_validation_tools/table-schemas/level-2/imc.yaml

```diff
@@ -99,8 +99,7 @@ fields:
     enum:
       - um
 - name: segment_data_format
-  description: This refers to the data type, which is a float for the IMC counts.
-  type: number
+  description: This refers to the data type, which is a "Float" for the IMC counts.
```

Comment:
@cebriggs7135 It looks like you removed the `type` field. It was `number` but now there is no longer a type for `segment_data_format`.

Reply:
@ngehlenborg Yes because it actually is the word "Float", not a float number.

Reply:
@cebriggs7135 :
- I'll make this an enum: `float`, `integer`, `string` should cover it.